data-tool/flows/config.py

```diff
@@ -109,6 +123,12 @@ class _Config():  # pylint: disable=too-few-public-methods
     ACCOUNT_SVC_CLIENT_SECRET = os.getenv('ACCOUNT_SVC_CLIENT_SECRET')
     ACCOUNT_SVC_TIMEOUT = os.getenv('ACCOUNT_SVC_TIMEOUT')
 
+    # batch delete flow
+    BATCHES = int(os.getenv('BATCHES', sys.maxsize))
+    BATCH_SIZE = int(os.getenv('BATCH_SIZE', 300))
+    DELETE_AUTH_RECORDS = os.getenv('DELETE_AUTH_RECORDS').lower() == 'true'
+    DELETE_COLIN_RECORDS = os.getenv('DELETE_COLIN_RECORDS').lower() == 'true'
```

Comment:
I think maybe we should name this something else.  

For now it's strictly for deleting corp_processing records from the extract.

`DELETE_CORP_PROCESSING_RECORDS` or something...